app/views/candidate_interface/sign_in/new.html.erb

```diff
@@ -0,0 +1,18 @@
+<% content_for :title, t('authentication.sign_in.heading') %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+
+    <h1 class="govuk-heading-xl">Sign in</h1>
+
+    <p class="govuk-body-l">Enter the email address you used to register with, and we will send you a link to sign in.</p>
+
+    <%= form_with model: @candidate, url: candidate_interface_sign_in_path, builder: GOVUKDesignSystemFormBuilder::FormBuilder, html: { novalidate: true } do |f| %>
+      <%= f.govuk_error_summary %>
+
+      <%= f.govuk_text_field :email_address, label: { text: t('authentication.sign_up.email_address.label')}, hint_text: t('authentication.sign_up.email_address.hint_label'), type: :email %>
+
+      <%= f.submit t('authentication.sign_up.button'), class: 'govuk-button', 'data-module': 'govuk-button', 'data-prevent-double-click': true %>
```

Comment:
What is `data-prevent-double-click`? Part of the form elements library?

Reply:
it is part of the form builder:
https://github.com/DFE-Digital/govuk_design_system_formbuilder/blob/82bb7cbf440835338bab688b4af38d139816b9d4/lib/govuk_design_system_formbuilder/elements/submit.rb#L9